src/py42/exceptions.py

```diff
@@ -54,7 +55,13 @@ class Py42HTTPError(Py42ResponseError):
     """A base custom class to manage all HTTP errors raised by an API endpoint."""
 
     def __init__(self, exception, message=None):
-        message = message or u"Failure in HTTP call {}".format(str(exception))
+        if not message:
+            response_content = "\n Response content: {}".format(exception.response.text)
```

Comment:
I think we should remove the newline here and keep the message all on the same line.